src/Components/ArticleByTopic.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getArticlesFromApi } from "./Utils/api";
+
+const ArticleByTopic = (AllArticles) => {
+  const [articles, setArticles] = useState([]);
+  const { topic_slug } = useParams();
+  const [isLoading, setIsLoading] = useState(true);
+  const [err, setErr] = useState(null);
+
+  useEffect(() => {
+    getArticlesFromApi(topic_slug)
+      .then((articlesApi) => {
+        setArticles(articlesApi);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        setErr(`Not found`);
+      });
+  }, [topic_slug]);
+  if (err) return <p>{err}</p>;
+  if (isLoading) return <h1> loading</h1>;
+  return (
+    <div>
+      {articles
+        .filter((article) => article.topic === topic_slug)
+        .map((filteredPerson) => (
+          <li>
```

Comment:
Currently we're getting an error in the console `Warning: Each child in a list should have a unique "key" prop.`

When we use .map to generate multiple items, we need to include the key to whatever is being generated which in this case should be the `<li>` on line 28